pkg/controller/prpqr_reconciler/prpqr_reconciler.go

```diff
@@ -154,37 +154,42 @@ func (r *reconciler) reconcile(ctx context.Context, req reconcile.Request, logge
 
 // TODO: Currently we create a single dummy prowjob just for testing. The actual implementation
 // will be introduced in https://issues.redhat.com/browse/DPTP-2577
-func generateProwjobs(org, repo, branch, prpqrName, prpqrNamespace string) []prowv1.ProwJob {
-	labels := map[string]string{
-		v1.PullRequestPayloadQualificationRunLabel: prpqrName,
-	}
+func generateProwjobs(org, repo, branch, prpqrName, prpqrNamespace string, releaseJobSpec []v1.ReleaseJobSpec) []prowv1.ProwJob {
+	var ret []prowv1.ProwJob
+	for _, releaseJob := range releaseJobSpec {
```

Comment:
So one thing we need to record somewhere is the "immitated job name", which is inferred from the spec with `JobName`: 

https://github.com/openshift/ci-tools/blob/ac7aa8f84ac5086728cf8318d0aa58cae2841977/pkg/api/metadata.go#L66-L68

This value then needs to go to the `Status.Jobs.ReleaseJobName` field of the status. I think it would be appropriate if this method returned a `map[string]Prowv1.ProwJob` instead, and this name would be the key. I think it could also be sensible to put that name into a label.